src/providers/eth_provider/starknet/relayer.rs

```diff
@@ -0,0 +1,93 @@
+use crate::{
+    models::transaction::transaction_data_to_starknet_calldata,
+    providers::eth_provider::{
+        constant::{CHAIN_ID, RPC_CONFIG},
+        error::{SignatureError, TransactionError},
+        provider::EthApiResult,
+        starknet::kakarot_core::{starknet_address, EXECUTE_FROM_OUTSIDE},
+    },
+};
+use reth_primitives::TransactionSigned;
+use starknet::{
+    accounts::{Account, ExecutionEncoding, ExecutionV1, SingleOwnerAccount},
+    core::types::Felt,
+    providers::{jsonrpc::HttpTransport, JsonRpcClient},
+    signers::{LocalWallet, SigningKey},
+};
+use std::{env::var, ops::Deref, str::FromStr, sync::LazyLock};
+use tokio::sync::MutexGuard;
+
+/// Signer for all relayers
+static RELAYER_SIGNER: LazyLock<LocalWallet> = LazyLock::new(|| {
+    LocalWallet::from_signing_key(SigningKey::from_secret_scalar(
+        Felt::from_str(&var("RELAYER_PRIVATE_KEY").expect("missing relayer private key"))
+            .expect("failed to parse relayer private key"),
+    ))
+});
```

Comment:
```suggestion
        Felt::from_str(&var("RELAYER_PRIVATE_KEY").expect("Missing relayer private key"))
            .expect("Failed to parse relayer private key"),
    ))
});
```

Reply:
I like lowercase :/ I don't think there is a standard in rust

Reply:
Sure let us keep as is this was just because I saw other places with this is upper case but this is a detail. Keep the version you prefer :)